### ISSUE TITLE
The new Climate Data Store Beta (CDS-Beta) is now live...

### ISSUE THREAD

**Issue by ferguskeatinge** [bluegreen-labs/ecmwfr#133]
https://forum.ecmwf.int/t/the-new-climate-data-store-beta-cds-beta-is-now-live/3315

The Beta version of the climate store is now live, will this package be updating to utilize the new climate store?

**Comment by khufkens** [comment]
Way ahead of you but couldn't formally merge or release on CRAN as this was only informally released on the Copernicus ECMWF meeting in June.

This was staged here:
https://github.com/bluegreen-labs/ecmwfr/pull/131

Note that I still have to change the RStudio addin behaviour to match the new conversion routines.

So short answer, yes. Please give me some time to sort the conversions of the API call (some might translate from the old to the new system directly, but I know some won't).

**Comment by khufkens** [comment]
To install, install from the branch directly or wait for the CRAN update which should be there before the end of the month. I'll probably keep the old CDS in parallel because I'm fairly certain they will not be able to pull the plug outright on the old system a month from now (judging from the vibes on the forum - and people coming back to post holiday surprises etc).

**Comment by khufkens** [comment]
Tentative workflow / timeline here: #134

**Comment by ferguskeatinge** [comment]
I am seeing strange behavior with the new climate story API when using Python, netcdf structure differs from the old CDS.

**Comment by khufkens** [comment]
Sadly this is outside my control. I gave some strong worded feedback during the meeting as well as in a recent survey, regarding the support of netcdf.

If you don't mind you can open a new issue, with a worked/reproducible example (include the request). This way I can start documenting some of this to provide other users context (either in the issues or later in a vignette).

**Comment by khufkens** [comment]
You should be able to install version 2.0.0 from github (not on CRAN yet). Note that you should drop the use of the "user" in the `wf_request()` function, and create a new key (personal access token). previous requests should remain backwards compatible (by and large).
